tests/discovery/test_linkedin_links.py

```diff
@@ -17,12 +17,12 @@ def test_get_links(self):
 
     def test_links_linkedin(self):
         dir_path = os.path.dirname(os.path.realpath(__file__))
-        f = open(dir_path + "/test_linkedin_links.txt").read()
+        mock_response = open(dir_path + "/test_linkedin_links.txt").read()
```

Comment:
As you are using open() do not forget to tidy up after you are finished with the file with a .close()